tests/stub/iteration/scripts/v4x4/tx_pull_1_nested.script

```diff
@@ -31,14 +45,18 @@ S: RECORD ["1_1"]
        SUCCESS {}
     C: PULL {"n": 1, "qid": 1}
     S: RECORD ["1_2"]
+       SUCCESS {"has_more": true}
+    C: RUN "CYPHER NESTED" {} {}
+    S: SUCCESS {"fields": ["x"], "qid": 3}
+    C: PULL {"n": 1, "[qid]": 3}
+    S: RECORD ["3_1"]
+       SUCCESS {}
+    C: PULL {"n": 1, "qid": 1}
+    S: RECORD ["1_2"]
+       SUCCESS {}
```

Comment:
I think this branch is query 4 with `RECORD ["4_1"]`?

Reply:
Yes, working on it at the moment.

Reply:
It was meant to be 1_3 actually.